### ISSUE TITLE
"Current color temperature" readout is incorrect

### ISSUE THREAD

**Issue by tremby** [xflux-gui/xflux-gui#38]
In the middle of the day now my screen looks to be at its default temperature, yet the GUI's preferences window is telling me I've set the nighttime color temperature to 3400K, and the current color temperature is 3400K. It certainly isn't. If I click preview, it turns much warmer for a few seconds.

**Comment by ntc2** [comment]
Turns out that GUI element was just poorly worded, so I removed it: what it was trying to do was tell what your color temp setting had been when you opened the preferences menu. I have no idea why it did that -- I've been maintaining fluxgui but I did not create it -- but perhaps the intent had been to giver the current temp, or perhaps the wording was just poor. We'll probably never know ...

**Comment by Kilian** [comment]
That's a little dramatic ;) Anyway, it was indeed supposed to show current color temperature, but this isn't working correctly due to the communication between fluxgui and xflux not working consistently.

**Comment by ntc2** [comment]
@Kilian, speaking of the communication between fluxgui and xflux, is xflux documented anywhere? When working on https://github.com/xflux-gui/xflux-gui/issues/35 I failed to figure out the official way to exit xflux interactive / `--nofork` mode. The problem was that if I just sent it SIGINT or CTRL+C, it would die before applying my most recent color setting. In the end I just inserted a `sleep(1)` before killing xflux :P
